src/js/Domain/RacingInfoDomain.js

```diff
@@ -0,0 +1,47 @@
+class RacingInfoDomain {
+  static MIN_CARNAME_SIZE = 1;
+  static MAX_CARNAME_SIZE = 5;
+  #raceParticipateCar;
+  #raceForward = {};
+  #matchNumber;
```

Comment:
```suggestion
  static MIN_CARNAME_SIZE = 1;
  static MAX_CARNAME_SIZE = 5;
  
  #raceParticipateCar;
  #raceForward = {};
  #matchNumber;
```
이렇게 개행을 한 번 해주면 구분하기 더 좋을 것 같아요!